py-polars/src/series/aggregation.rs

```diff
@@ -6,6 +6,20 @@ use crate::PySeries;
 
 #[pymethods]
 impl PySeries {
+    fn any(&self) -> Option<bool> {
+        match self.series.dtype() {
+            DataType::Boolean => Some(self.series.bool().unwrap().any()),
+            _ => None,
```

Comment:
This should raise an error informing that we expect a bool here.

Reply:
@ritchie46 just pushed a new commit. I realized I was missing the `ignore_nulls=False` Kleene oiption so added that in as well.